index.js

```diff
@@ -49,10 +26,19 @@ function SlackBot(config) {
 }
 
 // add a command
-SlackBot.prototype.addCommand = function (commandName, desc, command) {
-  var split = Utils.splitCommand(commandName);
-  this[split.commandName] = command;
-  this.commands[split.commandName] = { args: split.args, desc: desc };
+SlackBot.prototype.addCommand = function (command, args, desc, callback) {
+  var realCallback = callback;
+  var realDesc = desc;
+  var realArgs = args;
+
+  if (arguments.length === 3) {
```

Comment:
Maybe add a comment here to indicate this handles the case where the command has no args?